src/GridFS/ReadableStream.php

```diff
@@ -164,6 +167,8 @@ public function readBytes(int $length): string
             return '';
         }
 
+        assert($this->buffer !== null);
```

Comment:
Is this necessary to silence a psalm error?

If `initBufferFromCurrentChunk()` doesn't return `false`, `$this->buffer` is initialized from `Binary::getData()` and should never be null. I'd expect psalm to pick up on that.

Reply:
Feel free to link the ticket here when you create it for cross-referencing purposes.